bpf/kmesh/workload/include/workload_common.h

```diff
@@ -32,5 +32,9 @@ typedef struct {
     __u32 ipv4;
     __u32 port;
 } address_t;
+
+typedef struct {
```

Comment:
Kernel-native mode may also use BYTES type. It is better to move the `struct bytes` to `bpf/include/common.h`.